### ISSUE TITLE
Looking for a maintainer

### ISSUE THREAD

**Issue by vors** [vors/jupyter-powershell#16]
Splitting conversation from #15 into its own thread here.

### Motivation

My original goal with the project was a proof of concept that PowerShell kernel for jupyter is doable with a simple input-output redirection. I took some shortcuts to implement it and since a lot of people found the project useful, it would make sense to start improving it and fix some rough edges. I personally don't have interest in pursuing this further, but I appreciate the surprising attention that project received. That's why I'd like to facilitate a process of finding a new maintainer for the project. I'm looking for a single person, since it seems unnecessary to involve more than 1 person as a maintainer at this time. 

### Responsibilities

Maintainer should drive the project. They would receive the write access to repository. Maintainer is expected to react to incoming github issues and working strategically to improve the experience of using the kernel, starting with fixing existing open issues.

### Requirements 

If you are interested in driving this project forward, please first become a contributor. Issues #10 #15 are good ones to fix, but you can also take a peak at other open issues.

### My role

I'm keeping admin role for the project with a right to revoke maintainer's write access to the project. Examples of when that could happen:
1) Inappropriate behavior in github issues tracker
2) Long time inactivity in the project

### How to apply

1) Become a contributor! Send some pull requests!
2) Write a comment in this issue and express your interest. It would help to know how do you use this project to understand your long-term interes.

**Comment by TRM13** [comment]
Sorry for not getting back to you sooner. Vacation and off line for 2 weeks. 

I am interested but have very done little programming (my background is mostly sysadmin & scripting). I would be very rough and my time is limited. I use this to teach Powershell scripting (and Bash) plus I plan to eventually include Python but I'd need to learn it first. 

I am making up Jupyter Lab textbooks for my students for Powershell & Bash along with trying to integrate both Hub & NBgrader so my plate is rather full. I do get 2 months a year vacation but during the time I'm teaching (Nov-Aug) I'd be too busy.

**Comment by vors** [comment]
Alright, @TRM13 want to chime in #17 ? maybe we can figure out with combined brain-power how to find out the number of distinct commands in a cell or maybe find a way to produce only a single char :)

**Comment by vors** [comment]
We need to have a new release #18 . If somebody wants to figure and document the process that would be awesome. (I did it many years ago last time, so don't remember anything).

**Comment by vors** [comment]
@chlafreniere would you be interested in it?

**Comment by chlafreniere** [comment]
Absolutely!

**Comment by vors** [comment]
@chlafreniere done! You are the official maintainer of this project, **congratulations! 🎉** 

Couple pearls of wisdom:

## git

Not sure how comfortable you are with git.
Git hygiene (clear commits, easy to read history) is my pet peeve.
I'd very much appreciate, if you follow this guidelines:

- Keep master clean and free from merges
- Use rebase and squash button for PR merges
- create tags for releases
- write good commit messages and changelog entrances

## git resources

There are million ways to use git branching.
My current philosophy is the following: https://www.endoflineblog.com/gitflow-considered-harmful
Basically, dead-simple. One `master` branch. `rebase` all the things, never `merge`.

If you want to become git guru, this is my favorite way! https://github.com/Gazler/githug
It's an interactive GAME, and it's pretty addictive. It walks you thru the puzzles with git with increased difficulties. Something like 4 hours of your time and you will be the master of it.


You can always tag me if you have any questions.
